Covinator/Covinator/Pages/PfizerVaccination.cshtml.cs

```diff
@@ -45,10 +46,9 @@ public void OnGet()
 
                     string result_string = result_array.ToString();
 
-                    var pfizerVaccineDistributionAllocations = PfizerVaccineDistributionAllocations.FromJson(result_string);
+                    PfizerVaccineDistributionAllocations [] pfizerAllocations = PfizerVaccineDistributionAllocations.FromJson(result_string);
```

Comment:
changing generic datatype var to a more specific data type (array). var should not be used when datatype is not apparent.
Source:- https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/inside-a-program/coding-conventions